### ISSUE TITLE
[Bug] Auto-close during search/global search in catalogues

### ISSUE THREAD

**Issue by Mitia99** [inorichi/tachiyomi-extensions#2481]
### Device information
* Tachiyomi version: 0.8.5
* Android version: 9

## Steps to reproduce
1.Search a manga in japscan
2.Global catalogues search a manga with japscan extension

### Expected behavior
Display the searched manga and don't close Tachiyomi

### Actual behavior
Tachioymi auto-close

**Comment by arkon** [comment]
Can't seem to reproduce this. The app shouldn't crash even if it extension throws an exception when searching. 

Can you get the logs?

**Comment by Mitia99** [comment]
@arkon  Can't get the logs.Apps is working properly when I disable japscan extension.

**Comment by arkon** [comment]
Can you get the logs? Otherwise we can't fix it.

**Comment by Mitia99** [comment]
Where can I get the logs?

**Comment by arkon** [comment]
There's some guides online on how to get logcats. I'm not sure what the easiest way for non-developers is.

**Comment by Adam935** [comment]
Hello I also have the same problem I don’t know if these are the good logs but here.
Tell me if is not this 
[Logtest.txt](https://github.com/inorichi/tachiyomi-extensions/files/4398361/Logtest.txt)

**Comment by Adam935** [comment]
[Test_video.zip](https://github.com/inorichi/tachiyomi-extensions/files/4398377/Test_video.zip)
if you want to see

**Comment by arkon** [comment]
```
03-29 07:34:27.835 11421 11476 E ACRA    : ACRA caught a IllegalStateException for eu.kanade.tachiyomi
03-29 07:34:27.835 11421 11476 E ACRA    : java.lang.IllegalStateException: Fatal Exception thrown on Scheduler.Worker thread.
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:59)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at java.lang.Thread.run(Thread.java:919)
03-29 07:34:27.835 11421 11476 E ACRA    : Caused by: java.lang.NoSuchMethodError: No static method parseString(Ljava/lang/String;)Lcom/google/gson/JsonElement; in class Lcom/google/gson/JsonParser; or its super classes (declaration of 'com.google.gson.JsonParser' appears in base.apk)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at eu.kanade.tachiyomi.extension.fr.japscan.Japscan.searchMangaParse(Japscan.kt:186)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at eu.kanade.tachiyomi.source.online.HttpSource$fetchSearchManga$1.call(HttpSource.kt:122)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at eu.kanade.tachiyomi.source.online.HttpSource$fetchSearchManga$1.call(HttpSource.kt:23)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:69)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:101)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1$requestArbiter$1.request(OkHttpExtensions.kt:25)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber$1.request(OperatorSubscribeOn.java:109)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.producers.ProducerArbiter.setProducer(ProducerArbiter.java:126)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4.setProducer(OperatorOnErrorResumeNextViaFunction.java:159)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.setProducer(OperatorSubscribeOn.java:105)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.Subscriber.setProducer(Subscriber.java:205)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeMap$MapSubscriber.setProducer(OnSubscribeMap.java:102)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.Subscriber.setProducer(Subscriber.java:205)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:45)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:13)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.Observable.unsafeSubscribe(Observable.java:10327)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.Observable.unsafeSubscribe(Observable.java:10327)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.Observable.unsafeSubscribe(Observable.java:10327)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeDefer.call(OnSubscribeDefer.java:51)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OnSubscribeDefer.call(OnSubscribeDefer.java:35)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.Observable.unsafeSubscribe(Observable.java:10327)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.call(OperatorSubscribeOn.java:100)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:230)
03-29 07:34:27.835 11421 11476 E ACRA    : 	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
```

**Comment by Mitia99** [comment]
Couldn't find the way for log upload but I notice that the bug has been
fixed after 1.2.19 Japscan extension update.Thanks guys.
